core/src/edu/asu/c3simulator/screens/MarketingScreen.java

```diff
@@ -141,25 +140,10 @@ public MarketingScreen(Game game)
 	 * @return the NavigationPanel actor that was created.
 	 */
 	@Override
-	public NavigationPanel createNavigationPanel()
+	public void createNavigationPanel()
 	{
-		NavigationPanel navigationPanel = new NavigationPanel(game, skin);
-		
-		// TODO add screens
-		navigationPanel.addButton("Products", null);
-		navigationPanel.addSubButton("Products", "Pre-Market", null);
-		navigationPanel.addSubButton("Products", "Current", null);
-		navigationPanel.addSubButton("Products", "Retired", null);
-		navigationPanel.addButton("Product Growth", null);
-		navigationPanel.addSubButton("Product Growth", "Supply", null);
-		navigationPanel.addSubButton("Product Growth", "Demand", null);
-		navigationPanel.addButton("Employment", null);
-		navigationPanel.addButton("Marketing", null);
-		
-		navigationPanel.setPosition(0.01f * stage.getWidth(), stage.getHeight()
-				- (0.3f * stage.getHeight()));
-		
-		return navigationPanel;
+		@SuppressWarnings("unused")
+		ManagePanel navigation = new ManagePanel(game, stage);
```

Comment:
Tonight, on 10 o'clock news central, we cover the serial killer that keeps bringing these data stores to their grave. Detective C3 investigates.